<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="windowFilePath">
   <string/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="sideBar" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>120</width>
      <height>450</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
	background-color: rgb(228, 228, 228);
}</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="3" column="0">
      <widget class="QToolButton" name="btnSettring">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">/* 默认 */
        QToolButton{
              border-top: 3px outset transparent;
              border-bottom: 7px outset transparent;
              border-right: 3px outset transparent;
              border-left: 20px outset transparent;

              background-color: rgb(228, 228, 228);
        }

        /* 鼠标悬停 */
        QToolButton:hover{
              background-color: rgb(205, 205, 205);
        }

        /* 点击和按下 */
        QToolButton:pressed,QToolButton:checked{
              border-left: 18px outset rgb(93, 95, 97);
              background-color: rgb(246, 246, 246);
        }

        QPushButton:default {
          border-color: navy; /* make the default button prominent */
        }
        </string>
       </property>
       <property name="text">
        <string>   设置</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/ico/设置.png</normaloff>:/ico/设置.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="2">
      <widget class="QToolButton" name="btnHome">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">/* 默认 */
        QToolButton{
              border-top: 3px outset transparent;
              border-bottom: 7px outset transparent;
              border-right: 3px outset transparent;
              border-left: 20px outset transparent;

              background-color: rgb(228, 228, 228);
        }

        /* 鼠标悬停 */
        QToolButton:hover{
              background-color: rgb(205, 205, 205);
        }

        /* 点击和按下 */
        QToolButton:pressed,QToolButton:checked{
              border-left: 18px outset rgb(93, 95, 97);
              background-color: rgb(246, 246, 246);
        }

        QPushButton:default {
          border-color: navy; /* make the default button prominent */
        }
        </string>
       </property>
       <property name="text">
        <string>   主页</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/ico/主页.png</normaloff>:/ico/主页.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>100</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <widget class="QToolButton" name="btnMonitor">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">/* 默认 */
        QToolButton{
              border-top: 3px outset transparent;
              border-bottom: 7px outset transparent;
              border-right: 3px outset transparent;
              border-left: 20px outset transparent;

              background-color: rgb(228, 228, 228);
        }

        /* 鼠标悬停 */
        QToolButton:hover{
              background-color: rgb(205, 205, 205);
        }

        /* 点击和按下 */
        QToolButton:pressed,QToolButton:checked{
              border-left: 18px outset rgb(93, 95, 97);
              background-color: rgb(246, 246, 246);
        }

        QPushButton:default {
          border-color: navy; /* make the default button prominent */
        }
        </string>
       </property>
       <property name="text">
        <string>   监控</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/ico/监控.png</normaloff>:/ico/监控.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QToolButton" name="btnAbout">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">/* 默认 */
        QToolButton{
              border-top: 3px outset transparent;
              border-bottom: 7px outset transparent;
              border-right: 3px outset transparent;
              border-left: 20px outset transparent;

              background-color: rgb(228, 228, 228);
        }

        /* 鼠标悬停 */
        QToolButton:hover{
              background-color: rgb(205, 205, 205);
        }

        /* 点击和按下 */
        QToolButton:pressed,QToolButton:checked{
              border-left: 18px outset rgb(93, 95, 97);
              background-color: rgb(246, 246, 246);
        }

        QPushButton:default {
          border-color: navy; /* make the default button prominent */
        }
        </string>
       </property>
       <property name="text">
        <string>   关于</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/ico/关于.png</normaloff>:/ico/关于.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
       <property name="arrowType">
        <enum>Qt::NoArrow</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QToolButton" name="toolButton_close">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resources/qrc/ico.qrc">
      <normaloff>:/side_bar_plus/resources/images/关闭.png</normaloff>:/side_bar_plus/resources/images/关闭.png</iconset>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_min">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="../resources/qrc/ico.qrc">
      <normaloff>:/side_bar_plus/resources/images/横杠.png</normaloff>:/side_bar_plus/resources/images/横杠.png</iconset>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>680</width>
      <height>450</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources/qrc/ico.qrc"/>
 </resources>
 <connections/>
</ui>
